plugins/linguist/src/components/LinguistCard/LinguistCard.tsx

```diff
@@ -70,7 +73,7 @@ export const LinguistCard = () => {
 
   if (items && items.languageCount === 0 && items.totalBytes === 0) {
     return (
-      <InfoCard title="Languages" className={classes.infoCard}>
+      <InfoCard title={t('component.title')} className={classes.infoCard}>
```

Comment:
Can we also do line 80 that says: "There is currently no language data for this entity."? Feels like an easy one to include in this change